### PR TITLE
update sender vars when reloading config

### PR DESCRIPTION
**Pull request by SherinV**
Signed-off-by: Sherin Varughese <shvarugh@redhat.com>

**Related Issue:**  open-cluster-management/backlog#19855

### CODE REVIEW

File: pkg/send/sender.go [hunk 1]
```diff
@@ -84,6 +84,15 @@ type Sender struct {
 	rec                *reconciler.Reconciler
 }
 
+func (s *Sender) reloadSender() {
+	s.aggregatorURL = config.Cfg.AggregatorURL
+	s.aggregatorSyncPath = strings.Join([]string{"/aggregator/clusters/", config.Cfg.ClusterName, "/sync"}, "")
+	s.httpClient = getHTTPSClient()
```
Comment (jlpadilla) [inline pkg/send/sender.go:90]: Maybe move this line to the end?
```
s.httpClient = getHTTPSClient()
```

Comment (SherinV) [reply pkg/send/sender.go]: Done

File: pkg/send/sender.go [hunk 2]
```diff
@@ -169,7 +178,7 @@ func (s *Sender) sendWithRetry(payload Payload, expectedTotalResources int, expe
 			glog.Warningf("Received error response [%s] from Aggregator. Resending in %d ms after resetting config.",
 				sendError.Error(), waitMS)
 			config.InitConfig() // re-initialize config to get the latest certificate.
-			s.httpClient = getHTTPSClient()
+			s.reloadSender()    // reload sender variables - Aggregator URL, path and client
 			time.Sleep(time.Duration(waitMS) * time.Millisecond)
```
[the remark this answers is not in the document]
Comment (SherinV) [reply pkg/send/sender.go]: Done

**Pull request merged**